applications/nrf5340_audio/doc/feature_support.rst

```diff
@@ -21,7 +21,19 @@ For an explanation of the maturity levels, see :ref:`Software maturity levels <s
 nRF5340 Audio QDIDs
 *******************
 
-The following QDIDs are related to the nRF5340 Audio application:
+The following DNs/QDIDs are related to the nRF5340 Audio application:
+
+nRF5340 DK Bluetooth DNs/QDIDs
+   A full Audio product QDID requires several DNs and QDIDs for the nRF5340 DK, as listed in the `nRF5340 DK Bluetooth DNs and QDIDs Compatibility Matrix`_.
+   The exact DN/QDID numbers depend on the project configuration and the features used in the application.
```

Comment:
I find this (lines 28 - 30) inconsistent with the "A full Audio product" above and a bit confusing.
I am not  sure what we are trying to say here.